### PR TITLE
Updating bastion hubble host to incorporate data team members.

### PR DESCRIPTION
**Pull request by sheyd**
#### What's this PR do?
* Updates Hubble bastion host to use latest security practices and http/2 protocol.
* Corrects syntax for variable evaluation of `open_access`.
* Adds `open_access` users to Hubble bastion host.

#### Where should the reviewer start?
Below!
#### How should this be manually tested?
Already tested and deployed to Hubble Prod.
#### What are the relevant tickets?
#107

### CODE REVIEW

**Comment by sheyd** [comment]
@DFurnes ?

File: ansible/hubble-bastion.yml [hunk 1]
```diff
@@ -3,7 +3,9 @@
   become: true
   roles:
     - base-os
-    - nginx
+    - nginx-secure
+  vars:
+    open_access: true
```
Comment (DFurnes) [inline ansible/hubble-bastion.yml]: Not sure if this is a standard Ansible thing, but if not would it make sense to add a comment explaining what this does with regards to creating users and SSH keys below?

Comment (sheyd) [inline ansible/hubble-bastion.yml]: Notes are usually left in Ansible role, but comments definitely can be added to playbooks. I'll clean-up and add.

**Comment by DFurnes** [comment]
👍

File: ansible/hubble-bastion.yml (hunk 1 above)
Comment (DFurnes) [inline ansible/hubble-bastion.yml]: Cool cool. 👍 ✏️

**Pull request merged**